release.json

```diff
@@ -40,6 +40,9 @@
         "MACOS_BUILD_VERSION": "7.23.0",
         "SECURITY_AGENT_POLICIES_VERSION": "v0.5"
     },
+    "dca-1.9.0": {
+        "SECURITY_AGENT_POLICIES_VERSION": "v0.3"
```

Comment:
Out of curiosity more than anything: why v0.3 and not v0.5 like on 6.23.0/7.23.0?

Reply:
I had built the last RC when there was the decision on moving to 0.4 and then 0.5.
We did not test past 0.3 in the DCA although got confirmation that there would were no changes pertaining to the Cluster Agent between 0.3 and 0.5. I still decided to ship the current RC instead of building a new one just because the agent had 0.5.0

Reply:
Makes sense, thanks for the reply